charts/argo/values.yaml

```diff
@@ -51,6 +51,9 @@ ui:
   serviceAccount: argo-ui
   # Annotations to be applied to the UI Service
   serviceAnnotations: {}
+  # Static IP address to assign to loadBalancer
+  # service type `LoadBalancer`
+  loadBalancerIP: "123"
```

Comment:
I think this should be removed, otherwise it'll default to "123".

Reply:
Ah, yeah I tested if it works and forgot it there :+1: